gooddata-sdk/gooddata_sdk/catalog/identifier.py

```diff
@@ -82,7 +85,7 @@ def client_class() -> builtins.type[DeclarativeUserIdentifier]:
 @attr.s(auto_attribs=True, kw_only=True)
 class CatalogLabelIdentifier(Base):
     id: str
-    type: str = attr.field(validator=value_in_allowed)
+    type: str = attr.field(default="analyticalDashboard", validator=value_in_allowed)
```

Comment:
Is the default correct here? If I check [LabelIdentifier](https://github.com/gooddata/gooddata-python-sdk/blob/3f112f4641aaa27b13d1f60a4ef5ab9e57539897/gooddata-api-client/gooddata_api_client/model/label_identifier.py#L58-L58) client class there is only `label` in allowed_values.

Reply:
This looks like it was introduced when merging from latest and resolving conflicts. I did not modify this, let me revert.